modules/k6/k6_test.go

```diff
@@ -24,19 +27,42 @@ func TestK6(t *testing.T) {
 			script: "fail.js",
 			expect: 108,
 		},
+		{
+			title:  "Passing remote test",
+			script: "https://raw.githubusercontent.com/testcontainers/testcontainers-go/main/modules/k6/scripts/pass.js",
+			expect: 0,
+		},
+		{
+			title:  "Failing remote test",
+			script: "https://raw.githubusercontent.com/testcontainers/testcontainers-go/main/modules/k6/scripts/fail.js",
+			expect: 108,
+		},
 	}
 
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.title, func(t *testing.T) {
 			ctx := context.Background()
 
-			absPath, err := filepath.Abs(filepath.Join("scripts", tc.script))
-			if err != nil {
-				t.Fatal(err)
+			var options testcontainers.CustomizeRequestOption
+			if !strings.HasPrefix(tc.script, "http") {
+				absPath, err := filepath.Abs(filepath.Join("scripts", tc.script))
+				if err != nil {
+					t.Fatal(err)
+				}
+				options = k6.WithTestScript(absPath)
+			} else {
+
+				uri, err := url.Parse(tc.script)
```

Comment:
Parsing validation done here. I don't have a test for this portion, but I think we can assume url.Parse works.